tests/e2e/specs/out-of-game/home.spec.js

```diff
@@ -306,6 +314,119 @@ describe('Home - Game List', () => {
+
+      //Navigate to homepage and select spectate tab
+      cy.reload();
+      cy.get('[data-cy-game-list-selector=spectate]').click();
+
+      //Finished by stalemate
+      cy.createGameOpponent('Stalemate').then(({ gameId }) => {
+        recoverOpponentsInNewGame(gameId);
+        cy.stalemateOpponent();
+        cy.recoverSessionOpponent(playerOne);
+
+        //make sure button is still enabled after 1 person stalemates
+        cy.get(`[data-cy-spectate-game=${gameId}]`).should('be.not.disabled');
+        cy.stalemateOpponent();
+
+        cy.get(`[data-cy-spectate-game=${gameId}]`).should('be.disabled');
+      });
+
+      cy.reload();
+      cy.get('[data-cy-game-list-selector=spectate]').click();
+
+      //Finished by pass
+      cy.setupGameAsSpectator();
+      cy.loadGameFixture({
+        p0Hand: [Card.SEVEN_OF_CLUBS],
+        p0Points: [Card.SEVEN_OF_DIAMONDS, Card.SEVEN_OF_HEARTS],
+        p0FaceCards: [],
+        p1Hand: [],
+        p1Points: [],
+        p1FaceCards: [],
+      });
+      cy.get('[data-player-hand-card]').should('have.length', 1);
+      cy.log('Fixture loaded');
+      //Delete deck, draw last two card, both players pass until stalemate
+      cy.deleteDeck();
+      cy.log('Drawing last two cards');
+      cy.get('#deck').should('contain', '(2)');
+      cy.recoverSessionOpponent(playerOne);
+      cy.drawCardOpponent();
+      cy.get('#deck').should('contain', '(1)');
+      cy.recoverSessionOpponent(playerTwo);
+      cy.drawCardOpponent();
+      cy.get('#deck').should('contain', '(0)');
+      cy.log('Deck empty');
+      cy.recoverSessionOpponent(playerOne);
+      cy.passOpponent();
+      cy.get('#history').should('contain', `${playerOne.username} passes`);
+      cy.recoverSessionOpponent(playerTwo);
+      cy.passOpponent();
+      cy.get('#history').should('contain', `${playerTwo.username} passes`);
+      cy.recoverSessionOpponent({ username: playerUsername, password: playerPassword });
+      cy.vueRoute('/');
+      cy.get('[data-cy-game-list-selector=spectate]').click();
+      cy.get(`[data-cy-spectate-game]`).should('be.not.disabled');
+      cy.recoverSessionOpponent(playerOne);
+      cy.passOpponent();
+      cy.get(`[data-cy-spectate-game]`).should('be.disabled');
+
+      //Finished by P0 Victory
+      cy.setupGameAsSpectator();
+      cy.loadGameFixture({
+        p0Hand: [Card.EIGHT_OF_SPADES],
+        p0Points: [Card.SEVEN_OF_SPADES, Card.SEVEN_OF_HEARTS],
+        p0FaceCards: [],
+        p1Hand: [Card.EIGHT_OF_HEARTS],
+        p1Points: [Card.SEVEN_OF_DIAMONDS, Card.SEVEN_OF_CLUBS],
+        p1FaceCards: [],
+      });
+      cy.get('[data-player-hand-card]').should('have.length', 1);
+      cy.recoverSessionOpponent(playerOne);
+      cy.vueRoute('/');
+      cy.get('[data-cy-game-list-selector=spectate]').click();
+      cy.get(`[data-cy-spectate-game]`).should('be.not.disabled');
+      cy.playPointsSpectator(Card.EIGHT_OF_SPADES, 0);
+      cy.log('playedPoints');
```

Comment:
Let's get rid of all the cy.log() calls that are here and instead add two cy.log calls per  test situation (before and after) the case worked successfully e.g. 

```
      cy.log('Setup game end by passing')
      cy.setupGameAsSpectator();
      cy.loadGameFixture({
        p0Hand: [Card.SEVEN_OF_CLUBS],
        p0Points: [Card.SEVEN_OF_DIAMONDS, Card.SEVEN_OF_HEARTS],
        p0FaceCards: [],
        p1Hand: [],
        p1Points: [],
        p1FaceCards: [],
      });
      cy.get('[data-player-hand-card]').should('have.length', 1);
      //Delete deck, draw last two card, both players pass until stalemate
      cy.deleteDeck();
      cy.get('#deck').should('contain', '(2)');
      cy.recoverSessionOpponent(playerOne);
      cy.drawCardOpponent();
      cy.get('#deck').should('contain', '(1)');
      cy.recoverSessionOpponent(playerTwo);
      cy.drawCardOpponent();
      cy.get('#deck').should('contain', '(0)');
      cy.log('Deck empty');
      cy.recoverSessionOpponent(playerOne);
      cy.passOpponent();
      cy.get('#history').should('contain', `${playerOne.username} passes`);
      cy.recoverSessionOpponent(playerTwo);
      cy.passOpponent();
      cy.get('#history').should('contain', `${playerTwo.username} passes`);
      cy.recoverSessionOpponent({ username: playerUsername, password: playerPassword });
      cy.vueRoute('/');
      cy.get('[data-cy-game-list-selector=spectate]').click();
      cy.get(`[data-cy-spectate-game]`).should('be.not.disabled');
      cy.recoverSessionOpponent(playerOne);
      cy.passOpponent();
      cy.get(`[data-cy-spectate-game]`).should('be.disabled');
      cy.log('Game ended by passing - successfully disabled spectate button');
```

This will clean up the log output and focus the logs on the outcome of a test case in broad strokes